facturacion_mexico/complementos_pago/api.py

```
@@ -174,7 +174,32 @@ def timbrar_complemento_pago(complemento_name: str) -> dict:
+			flag = frappe.db.get_value("Factura Fiscal Mexico", ffm_name, "fm_facturar_venta_mostrador") or 0
+			venta_mostrador_flags.add(int(flag))
+
+		if len(venta_mostrador_flags) > 1:
+			frappe.throw(
+				_(
+					"Las facturas de este pago tienen configuraciones de receptor distintas "
+					"(mezcla de Venta Mostrador y cliente normal). Separe el pago."
+				)
+			)
+		if venta_mostrador_flags == {1}:
+			if frappe.db.exists("Customer", "VENTA MOSTRADOR"):
```

Ruff (0.15.12)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)